packages/react-router-config/modules/__tests__/integration-test.js

```diff
@@ -16,9 +16,11 @@ describe('integration', () => {
     const routes = [
       { path: '/pepper',
         component: Comp,
+        parent: true,
```

Comment:
`react-router-config` should be able to just set the `parent` property based on whether the `routes` array exists and has a length, but that can be changed in another PR.